src/components/DocumentViewer.tsx

```diff
@@ -1,22 +1,36 @@
-// @ts-ignore
-import React, { useRef, useEffect } from 'react'
-import useInstances from '../context'
+import React from 'react';
+import useInstance from '../context';
+
+const disabledElements = [
+  'header',
+  'toolsHeader',
+  'viewControlsButton',
+  'leftPanelButton',
+  'pageNavOverlay',
+  'searchButton',
+  'menuButton',
+  'textPopup',
+  'contextMenuPopup',
+];
```

Comment:
Please refer to the specification I gave you when starting this project. This component is supposed to just be a normal Viewer and should not hide any elements. Instead, we want a separate "SimpleViewer" component that does this stuff

Reply:
I have moved these options into util.ts. It is imported in the simple viewer.